metrics/web/test/dashboard/presentation/widgets/build_result_popup_card_build_status_test.dart

```diff
@@ -0,0 +1,246 @@
+      },
+    );
+
+    testWidgets(
+      "applies the right animation asset to the RiveAnimationWidget",
+      (WidgetTester tester) async {
+        const expectedAsset =
+            'web/animation/in_progress_popup_build_status.riv';
+
+        await tester.pumpWidget(
+          const _BuildResultPopupCardBuildStatusTestbed(
+            projectBuildStatus: inProgressProjectBuildStatus,
+          ),
+        );
+
+        final riveAnimation = tester.widget<RiveAnimation>(
+          riveAnimationFinder,
+        );
+
+        expect(riveAnimation.assetName, equals(expectedAsset));
+      },
+    );
+
+    testWidgets(
+      "applies the true 'use artboard size' parameter to the RiveAnimation widget",
+      (WidgetTester tester) async {
+        await tester.pumpWidget(
+          const _BuildResultPopupCardBuildStatusTestbed(
+            projectBuildStatus: inProgressProjectBuildStatus,
+          ),
+        );
+
+        final riveAnimation = tester.widget<RiveAnimation>(
+          riveAnimationFinder,
+        );
+
+        expect(riveAnimation.useArtboardSize, isTrue);
+      },
+    );
+
+    testWidgets(
+      "applies the animation controller with the correct animation name to the RiveAnimation widget",
+      (WidgetTester tester) async {
+        const expectedAnimationName = 'Animation 1';
+
+        await tester.pumpWidget(
+          const _BuildResultPopupCardBuildStatusTestbed(
+            projectBuildStatus: inProgressProjectBuildStatus,
+          ),
+        );
+
+        final riveAnimation = tester.widget<RiveAnimation>(
+          riveAnimationFinder,
+        );
+
+        final animationControllerMatcher = predicate((controller) {
+          return controller is SimpleAnimation &&
+              controller.animationName == expectedAnimationName;
+        });
+
+        expect(riveAnimation.controller, equals(animationControllerMatcher));
+      },
+    );
+
+    testWidgets(
+      "displays the ValueImage widget if the given project build status is finished",
+      (WidgetTester tester) async {
+        await mockNetworkImagesFor(() {
+          return tester.pumpWidget(
+            const _BuildResultPopupCardBuildStatusTestbed(
+              projectBuildStatus: successfulProjectBuildStatus,
+            ),
+          );
+        });
+
+        expect(valueImageFinder, findsOneWidget);
+      },
+    );
+
+    testWidgets(
+      "applies the builds status from the view model to the value network image",
+      (WidgetTester tester) async {
+        await mockNetworkImagesFor(() {
+          return tester.pumpWidget(
+            const _BuildResultPopupCardBuildStatusTestbed(
+              projectBuildStatus: successfulProjectBuildStatus,
+            ),
+          );
+        });
+
+        final valueNetworkImage = tester.widget<ValueNetworkImage>(
+          valueImageFinder,
+        );
+
+        expect(
+          valueNetworkImage.value,
+          equals(successfulProjectBuildStatus.value),
```

Comment:
```suggestion
          equals(expectedValue),
```
Introduce the expected value in the above as follows: 
```suggestion
          final expectedValue = successfulProjectBuildStatus.value;
```